docs/learn-staking.md

```diff
@@ -220,8 +245,8 @@ please see [here](https://research.web3.foundation/en/latest/polkadot/Token%20Ec
 
 ## Why not stake?
 
-* Tokens will be locked for 28 days after unbonding.
-* Tokens can be slashed as punishment if the validator is not running properly.
+* Tokens will be locked for about 12 weeks on Polkadot, seven days on Kusama
```

Comment:
"12 weeks on Kusama" -> is this set in stone?  I thought most Kusama params were set to 4x as fast as Polkadot.

Reply:
This was not my addition, I added the seven days. Let me check.

Reply:
Checked in code, it's 28 eras so 28 days.